src/ring_theory/finiteness.lean

```diff
@@ -144,6 +144,11 @@ end finite
 
 end module
 
+instance module.finite.tensor_product [comm_semiring R]
+  [add_comm_monoid M] [module R M] [add_comm_monoid N] [module R N]
+  [hM : module.finite R M] [hN : module.finite R N] : module.finite R (tensor_product R M N) :=
+{ out := (tensor_product.map₂_mk_top_top_eq_top R M N).subst (hM.out.map₂ _ hN.out) }
```

Comment:
Any reason to not put this above `end module` above so that you don't need the `module.`?

Reply:
Ah, that makes sense. It's fine here then